flatten-array.md

```diff
@@ -1,8 +1,11 @@
-The challenge is to write a function that accepts an arbitrarily-deep nested list-like structure and returns a flattened structure without any null/nill values. 
- 
-For Example
+The challenge is to write a function that accepts an arbitrarily-deep nested list-like structure and returns a flattened structure without any null/nill values.
+
+For example:
 
 input: [1,[2,3,null,4],[null],5]
 
 output: [1,2,3,4,5]
 
+## Note
+
+Implement the function yourself. Do not use something else to perform the flattening for you.
```

Comment:
And sometimes the opposite happens. People will implement things that the Common Lisp language has built in features for...